src/views/AccountCreation/index.tsx

```diff
@@ -24,8 +25,66 @@ interface Props {
   navigation: NavigationContainerRef
 }
 
+interface UserData {
+  mail: string
+  tag: string
+  name: string
```

Comment:
It is just a sugestion, but maybe you could put user picture in the same state ?

Reply:
Good idea 😀